tests/loggers/test_cometml_logger.py

```diff
@@ -1,18 +1,89 @@
 # Copyright 2022 MosaicML Composer authors
 # SPDX-License-Identifier: Apache-2.0
+import imghdr
 import os
 import zipfile
 from collections import defaultdict
 from json import JSONDecoder
 from pathlib import Path
+from typing import Sequence
 
 import pytest
+import torch
 from torch.utils.data import DataLoader
 
 from composer.trainer import Trainer
 from tests.common import RandomClassificationDataset, SimpleModel
 
 
+@pytest.fixture
+def comet_offline_directory(tmp_path):
+    return str(tmp_path / Path('.my_cometml_runs'))
+
+
+@pytest.fixture
+def comet_logger(monkeypatch, comet_offline_directory):
+    pytest.importorskip('comet_ml', reason='comet_ml is optional')
+    import comet_ml
+
+    monkeypatch.setattr(comet_ml, 'Experiment', comet_ml.OfflineExperiment)
+    from composer.loggers import CometMLLogger
+
+    # Set offline directory.
+    os.environ['COMET_OFFLINE_DIRECTORY'] = comet_offline_directory
+
+    comet_logger = CometMLLogger()
+    return comet_logger
+
+
+@pytest.mark.parametrize('images,channels_last', [(torch.rand(32), False), (torch.rand(32, 32), False),
```

Comment:
when would you have a 1D image?

Reply:
Visualizing a 1D heatmap of a representation vector or the output of a softmax.

Reply:
:+1: thanks